### ISSUE TITLE
Accuracy Statistics

### ISSUE THREAD

**Issue by lexNeurii** [DataTurks-Engg/Entity-Recognition-In-Resumes-SpaCy#3]
Hi,
Thanks for putting together this project. I have a question about the accuracy reporting. It seems that we are only reporting accuracy (and F1 scores etc ) only for the last Resume and not a aggregate scores. Is that a correct understanding?

**Comment by NishuKumar** [comment]
Yes correct..This only only shows measurement criteria for last resume.
